docs/production.md

```diff
@@ -1,60 +1,134 @@
 Install CommCare Sync for Production
-=====================
-This page outlines the process you need to follow in order to set up a CommCare Sync instance on a production machine.
+====================================
 
-## Choose a location for the control
+This page outlines the process you need to follow to set up a CommCare
+Sync instance on a production machine.
 
-While it is possible to run the ansible playbooks from anywhere, 
+## Choose a location for the control node
+
+While it is possible to run the Ansible playbooks from anywhere,
 it is recommended to run them *on the server you are setting up*.
-This will ensure consistency and also streamline the install process.
+This will ensure consistency and also streamline the installation
+process.
+
+These instructions assume a setup where the Ansible control node and
+playbooks are run on the server being set up.
+
+## Set up the server
+
+Dimagi installs CommCare Sync on Ubuntu 24.04 LTS.
+
+### SSH access
+
+Assuming you are running on AWS, copy the AWS private key to
+`~/.ssh/myproject.pem` on your local machine.
+
+You may also need to change permissions on it.
+
+```shell
+$ chmod 400 ~/.ssh/myproject.pem
+```
+
+Test it's working:
+
+```shell
+$ ssh -i ~/.ssh/myproject.pem ubuntu@myproject.example.com
+```
 
-These instructions assume a set up where the ansible control and playbooks are run on the server being set up.
+### Set the hostname
 
-## Install Ansible
+*On the remote server*
+
+```shell
+$ sudo hostnamectl set-hostname myproject.example.com`
+```
+
+### Install Ansible
 
-From the [Ansible Installation Guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-on-ubuntu),
+From the
+[Ansible Installation Guide](https://docs.ansible.com/projects/ansible/latest/installation_guide/installation_distros.html#installing-ansible-on-ubuntu),
 run the following on your control machine.
 
-```bash
-sudo apt-add-repository ppa:ansible/ansible
-sudo apt-get update
-sudo apt-get install ansible
+```shell
+$ sudo apt update
+$ sudo apt install software-properties-common
+$ sudo add-apt-repository --yes --update ppa:ansible/ansible
```

Comment:
Does Ubuntu 24.04 not have a reasonable version of Ansible in the default apt repositories?

Reply:
Good question. I'll check.